tests/Behat/features/preview-a-block.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Preview a non-inline-editable block
+  As an author
+  I want to preview the parent page of the non-inline-editable block I'm editing in the CMS
+  So that I can see how it would look like to my visitors
+
+  Background:
+    Given I add an extension "DNADesign\Elemental\Extensions\ElementalPageExtension" to the "Page" class
+      And content blocks are not in-line editable
+      And the page template includes elemental blocks
+      And a "page" "Blocks Page" with a "Alice's Block" content element with "Alice's content" content
+
+    Given I am logged in with "ADMIN" permissions
+      And I go to "/admin/pages"
+      And I left click on "Blocks Page" in the tree
+      And I set the CMS mode to "Edit mode"
+    Then I should see an edit page form
```

Comment:
Shouldn't be doing assertions in 'Background', that's really just a test setup bit

Reply:
Yeah I thought it was odd too - but this test is largely copying from other tests which also do this (see the edit block feature for example) - It's kinda good to know the setup is working as expected before we even get to the scenarios without having to repeat the same assertions again which is probably why it's done in the other ones.
That said I can change it if you feel strongly about it.

Reply:
Would prefer the assertions in the first scenario

Reply:
Fair. It does mean anyone running the second scenario locally won't have those assertions run but I can live with that. I'll make that change before pulling this out of draft.